test/Integration/test-csharp/GetProductColumnTypesSerializationAsyncEnumerable.cs

```diff
@@ -13,30 +13,35 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.Sql.Samples.InputBindingSamples
 {
-    public static class GetProductsColumnTypesSerializationDifferentCulture
+    public static class GetProductsColumnTypesSerializationAsyncEnumerable
     {
         /// <summary>
         /// This function verifies that serializing an item with various data types
-        /// works when the language is set to a non-enUS language.
-        /// Note this uses IAsyncEnumerable because IEnumerable serializes the entire table directly,
-        /// instead of each item one by one (which is where issues can occur)
+        /// and different languages works when using IAsyncEnumerable.
         /// </summary>
-        [FunctionName(nameof(GetProductsColumnTypesSerializationDifferentCulture))]
+        [FunctionName(nameof(GetProductsColumnTypesSerializationAsyncEnumerable))]
         public static async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "getproducts-columntypesserializationdifferentculture")]
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "getproducts-columntypesserializationasyncenumerable")]
             HttpRequest req,
             [Sql("SELECT * FROM [dbo].[ProductsColumnTypes]",
                 CommandType = System.Data.CommandType.Text,
                 ConnectionStringSetting = "SqlConnectionString")]
             IAsyncEnumerable<ProductColumnTypes> products,
             ILogger log)
         {
-            CultureInfo.CurrentCulture = new CultureInfo("it-IT", false);
+            string language = req.Query["culture"];
+            if (string.IsNullOrEmpty(language))
```

Comment:
`!string.IsNullOrEmpty`

And add an explanation comment here about why we do this (basically summarize what I put before)